src/routes/contact-us/+page.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+	import fb_logo from './Facebook_Logo_Primary.png';
```

Comment:
JavaScript variables should be camel case.

Reply:
fixed in [5c83696](https://github.com/kendoclub-at-umich/website/pull/32/commits/5c83696a9b0f2457b89f3f46d87caf706c8f5940)